blocks/highlight-card/highlight-card.js

```diff
@@ -0,0 +1,30 @@
+export default function decorate(block) {
+  block.querySelectorAll('a').forEach((buttonLink) => {
+    buttonLink.classList.add('button');
+    buttonLink.classList.add('secondary');
+    buttonLink.closest('div').classList.add('button-container');
+  });
```

Comment:
Is it not possible for the author to control the button appearance? If they want to include a primary or secondary button they could just make the link **bold** or _italic_. Or are you trying to force a specific button type regardless of what the author provided?

Reply:
Forcing specific button type here.